.travis.yml

```diff
@@ -10,22 +10,6 @@ matrix:
     # Nightly builds run the tests/examples end-to-end test suite.
     # Does not include Windows since Projects/Models are mostly not compatible with Windows as of now.
     - stage: Nightly
-    - language: python
-      python: 2.7
-      install:
-        # CHANGED_FILES detects whether or not files related to the nightly build are changed.
-        # True is so that the command returns a 0 exit code whether or not CHANGED_FILES is empty.
-        - CHANGED_FILES=$(git diff --name-only master..HEAD | grep "tests/examples\|examples\|Dockerfile") || true;
-        - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || ! -z "$CHANGED_FILES" ]]; then
-            INSTALL_LARGE_PYTHON_DEPS=true source ./travis/install-common-deps.sh;
-          fi;
-      script:
-        - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$CHANGED_FILES" == *"examples"* ]]; then
-            pytest --verbose tests/examples --large;
-          fi
-        - if [[ "$TRAVIS_EVENT_TYPE" == "cron" || "$CHANGED_FILES" == *"Dockerfile"* ]]; then
-            docker build -t mlflow_test_build . && docker images | grep mlflow_test_build;
-          fi
```

Comment:
That's the nightly build. Are you they are run on build ?

Reply:
Yes, they are run at each build